sdk/nodejs/automation/events.ts

```diff
@@ -61,35 +61,38 @@ export interface PolicyEvent {
 }
 
 /**
- * PreludeEvent is emitted at the start of an update.
+ * An event emitted at the start of an update.
  */
 export interface PreludeEvent {
     /**
-     * config contains the keys and values for the update.
-     * Encrypted configuration values may be blinded.
+     * Configuration values that will be used during the update.
      */
     config: Record<string, string>;
 }
 
 /**
- * SummaryEvent is emitted at the end of an update, with a summary of the changes made.
+ * An event emitted at the end of an update, with a summary of the changes made.
  */
 export interface SummaryEvent {
     /**
-     * maybeCorrupt is set if one or more of the resources is in an invalid state.
+     * True if one or more of the resources is in an invalid state.
      */
     maybeCorrupt: boolean;
+
     /**
-     * duration is the number of seconds the update was executing.
+     * The number of seconds the update took to execute.
      */
     durationSeconds: number;
+
     /**
-     * resourceChanges contains the count for resource change by type. The keys are deploy.StepOp,
+     * The count for resource change by type. The keys are of type deploy.StepOp,
```

Comment:
```suggestion
     * The count for resource changes by type. The keys are of type deploy.StepOp,
```

The part about `deploy.StepOp` is confusing and no very helpful, maybe we should drop that?

Reply:
Will do :+1: